### ISSUE TITLE
Add new call to action to create New exercises pack

### ISSUE THREAD

**Issue by jorgegalindocruces** [scala-exercises/scala-exercises#506]
When the documentation about that will be completed, we have change the design in Home to add a box with a call to action to add the exercises of your own library.

**Comment by dialelo** [comment]
Depends on #120

**Comment by juanpedromoreno** [comment]
@jorgegalindocruces:

Is this issue already covered by https://github.com/scala-exercises/scala-exercises/pull/549?

Thanks.

**Comment by juanpedromoreno** [comment]
@jorgegalindocruces:

The link will be https://github.com/scala-exercises/scala-exercises/blob/master/DEV_GUIDE.md
once https://github.com/scala-exercises/scala-exercises/pull/549 is merged.

**Comment by raulraja** [comment]
#549 has been merged and the link is publicly available. @jorgegalindocruces we are ready for the frontend placeholder and link. Thanks!

**Comment by raulraja** [comment]
This was already addressed and it's now live in prod. 
![screen shot 2016-08-30 at 11 31 58 am](https://cloud.githubusercontent.com/assets/456796/18083889/68585886-6ea5-11e6-8ac8-cc2c49e62216.png)
